.github/workflows/test-ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     - name: Clean up dependencies
       run: |
         pip uninstall -y stable-baselines3 gymnasium
-        pip install .[rllib]
+        pip install ray[rllib]
```

Comment:
I think rllib requires gymnasium==0.26.3 which I why we install our .[rllib] deps, see: 
https://github.com/edbeeching/godot_rl_agents/blob/00ddb9e8b1cbe6264bf2de44eddc64dbe375f6ba/setup.cfg#L50